tests/unit/sinon-sandbox-test.js

```diff
@@ -18,6 +18,23 @@ test('stores sandbox created as module property', function(assert) {
   restoreSandbox();
 });
 
+test('stubbing out sandbox.create returns the already created sandbox', function(assert) {
+  assert.expect(1);
+
+  createSandbox();
+
+  const sandbox =  QUnit.config.current.testEnvironment.sandbox;
```

Comment:
You should be able to do `this.sandbox` here (since that is what `QUnit.config.current.testEnvironment.sandbox` is)...

Reply:
Yerp. Changing.